components/xpath/src/eval.rs

```diff
@@ -4,7 +4,7 @@
 
 use std::fmt;
 
-use html5ever::{LocalName, Namespace, Prefix, QualName, local_name, namespace_prefix, ns};
```

Comment:
I still think it might make sense to move `QualName` and `ExpandedName` into `web_atoms` so that libraries like this could just depend on that crate.

Reply:
Hmm maybe, though `markup5ever` is not that big of a dependency.